README.md

```diff
@@ -226,7 +226,8 @@ Dockerfile is supplied to build images with cuda support and cudnn v7. Build as
 docker build -t pytorch -f docker/pytorch/Dockerfile .
 ```
 
-Alternatively, if you want to use a runtime image, you can use the pre-built one from Docker Hub and run with nvidia-docker:
+You can also pull a pre-built docker image from Docker Hub and run with nvidia-docker,
+but this is not currently maintained and will pull PyTorch 0.2.
```

Comment:
Can we upload new images/remove the old ones? I doubt anyone wants 0.2 at this point

Reply:
Sure, but best to document it for now. Tracking issue #6397